department-of-veterans-affairs/vets-api
# Pension 71897 benefit intake polling

## Summary

- add BenefitIntake API polling db entries (FormSubmmission and FormSubmissionAttempt) on success
- Pensions

## Related issue(s)

https://github.com/department-of-veterans-affairs/va.gov-team/issues/71897

## Testing done

- No form_submission entry

- submit a 527ez (Pension) form locally
- run local vets-api rails console
  - `>  Lighthouse::PensionBenefitIntakeJob.new().perform(ID)` (ID = submitted form id in step above)
  - local database should have an entry in form_submission and form_submission_attempt
- `rspec spec/sidekiq/lighthouse/pension_benefit_intake_job_spec.rb`
  - all tests should pass
 
- same steps as above done as developer testing

## Acceptance criteria
